triton-vm/src/proof_item.rs

```diff
@@ -119,8 +119,7 @@ pub enum ProofItem {
 impl MayBeUncast for ProofItem {
     fn uncast(&self) -> Vec<BFieldElement> {
         if let Self::Uncast(vector) = self {
-            let mut str = vec![];
-            str.push(BFieldElement::new(vector.len().try_into().unwrap()));
+            let mut str = vec![BFieldElement::new(vector.len().try_into().unwrap())];
```

Comment:
This is probably an improvement but doesn't really belong to the current scope. I have a slight (but not strong) preference to do it in a separate cleanup commit.